### PR TITLE
Added new python resource (Real Python)

### PR DESCRIPTION
**Pull request by preethi1915**
## Fixes Issue
Related to #956 

## Changes proposed

Added 'Real Python' Learning Resource link
## Screenshots
Screenshot from 2023-06-12 19-11-39

### CODE REVIEW

File: database/languages/python.json [hunk 1]
```diff
@@ -26,5 +26,12 @@
         "url": "https://www.learnpython.org/",
         "category": "languages",
         "subcategory": "python"
+    },
+    {
+        "name": "Real Python Tutorial",
+        "description": "Learn Python through Real Hands on experience",
```
Comment (CBID2) [inline database/languages/python.json:32]: [Nitpick] This description could use more information. What other ways can developers and other tech professionals can benefit from this resource?
```suggestion
        "description": "Learn Python through Real Hands on experience",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @preethi1915! :) I gave a suggested on how you can improve your description.

File: database/languages/python.json (hunk 1 above)
Comment (preethi1915) [reply database/languages/python.json]: Hi @CBID2 , I have updated the description. Please have a look at it and let me know if any other changes are required:)

File: database/languages/python.json [hunk 2]
```diff
@@ -26,5 +26,12 @@
         "url": "https://www.learnpython.org/",
         "category": "languages",
         "subcategory": "python"
+    },
+    {
+        "name": "Real Python Tutorial",
+        "description": "Learn Python through Real Hands on experience. Learn Web development using Python, Data Science, Machine learning through well explained projects",
```
Comment (rupali-codes) [inline database/languages/python.json:32]:
```suggestion
        "description": "Learn Web development using Python, Data Science, Machine learning through well explained projects.",
```

File: database/languages/python.json (hunk 1 above)
Comment (CBID2) [reply database/languages/python.json]: Hi @preethi1915. There are some merge conflicts in your branch, so work on that. :)

**Review by rupali-codes — APPROVED** [review]
lgtm, please resolve merge conflicts

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**